packages/maskbook/src/plugins/Trader/UI/trending/TrendingPopper.tsx

```diff
@@ -90,7 +90,7 @@ export function TrendingPopper(props: TrendingPopperProps) {
                 anchorEl={anchorEl}
                 disablePortal
                 transition
-                style={{ zIndex: 1, margin: 4 }}
```

Comment:
The popper appeared under the Minds sidebar with a `zIndex: 1`. I thought it makes sense for the popper to be above other elements. But let me know if there's a special reason why this should be 1. I also tested whether the popper will inappropriately come over other elements such as dialogs, but couldn't find an issue.